packages/react/src/ProgressivelyProvider.tsx

```diff
@@ -16,8 +16,8 @@ export const ProgressivelyProvider = ({
   const [trackFn, setTrackFn] = useState<ProgressivelySdkType["track"]>(
     (eventName: string, data?: any) => Promise.resolve(undefined)
   );
-  const [isLoading, setIsLoading] = useState(true);
-  const [error, setError] = useState<any>();
+  const [status, setStatus] = useState<StateMachineConstants>("idle");
+  const [error, setError] = useState<any>(null);
```

Comment:
Nitpicky: we can leave it undefined by default, no need to add extra chars in the bundle :D